### ISSUE TITLE
when release update version to PYPI?

### ISSUE THREAD

**Issue by rexzhang** [maxcountryman/flask-themes#8]

**Comment by sbraz** [comment]
@maxcountryman Hi, when do you plan on making a new release? The old one is broken in all sorts of ways with recent flask versions.

**Comment by maxcountryman** [comment]
Well I’d like to release soon, however I’m busy and getting PyPI setup is kind of a pain. Also please bear in mind I don’t get paid for this work, so it’s unfortunately not a super high priority. In the meantime you can also install from git.

**Comment by sbraz** [comment]
Of course, I understand that. It's just that packaging the git version for us (Gentoo) isn't as straightforward as using a PyPI release (but it's doable :smiley:).

**Comment by maxcountryman** [comment]
That makes sense—I’ll try to get PyPI setup in the near future.

**Comment by sbraz** [comment]
@maxcountryman actually I might not need the new release so soon, I misunderstood the goal of the package, what I wanted is https://github.com/pallets/flask-sphinx-themes/.

I now understand that your package serves a wholly different purpose.
